solr/core/src/test/org/apache/solr/analysis/TestWordDelimiterFilterFactory.java

```diff
@@ -198,7 +199,7 @@ public void testCustomTypes() throws Exception {
     String testText = "I borrowed $5,400.00 at 25% interest-rate";
     ResourceLoader loader = new SolrResourceLoader(TEST_PATH().resolve("collection1"));
     Map<String,String> args = new HashMap<>();
-    args.put("luceneMatchVersion", Version.LATEST.toString());
+    args.put(IndexSchema.LUCENE_MATCH_VERSION_PARAM, Version.LATEST.toString());
```

Comment:
I'm going to remove these diffs in this file for consistency reasons.  All string literals is very clear.